run.py

```diff
@@ -44,6 +44,8 @@ def main(context):
             # otherwise, do nothing.
             if perp_char in ["z", "y"]:
                 adjustment_matrix[:, 0] = -1 * adjustment_matrix[:, 0]
+            if perp_char == "z":  # also flip slice direction
+                adjustment_matrix[:, 2] = -1 * adjustment_matrix[:, 2]
```

Comment:
The direction of this matrix may depend on `ImagePosition[N] - ImagePosition[1]` for a dicom. See that [Blind Study Stuff](https://github.com/flywheel-apps/nyu_rotator_cuff_gear_suite/blob/master/gears/gather_cases/utils/manage_cases.py#L213).

Yeah, this is me avoiding my work ;)...